docs/get-started/installing-cardano-node.md

```diff
@@ -89,23 +89,16 @@ You can check if `ghcup` has been installed correctly by typing `ghcup --version
 The GHCup Haskell installer, version v0.1.14.1
 ```
 
-`ghcup` will install the latest stable version of `ghc`. However, as of the time of writing this, [Input-Output](https://iohk.io) recommends using `ghc 8.10.4`. So, we will use `ghcup` to install and switch to the required version. 
-
-```bash
-ghcup install ghc 8.10.4
-ghcup set ghc 8.10.4
-```
-
 Finally, we check if we have the correct `ghc` and `cabal` versions installed.
 
 Check `ghc` version: 
 ```bash
 ghc --version
 ```
 
-You should see something like this: 
+You should see something like this (current versione 8.10.7): 
```

Comment:
Unless advised otherwise 8.10.4 is the version recommended in docs, while higher than 8.10.7 'should' work, unless the baseline docs are updated, no reason to change the ghc version here